packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -392,7 +392,7 @@ class PathColumn extends ColumnData<LinkData>
   }
 
   @override
-  String getValue(LinkData dataObject) => dataObject.path;
+  String getValue(LinkData dataObject) => dataObject.path ?? '';
```

Comment:
consider adding a getter on the LinkData class: `String get safePath = path ?? '';` and using that everywhere you are adding the `?? ''` logic